src/main/kotlin/club/sk1er/mods/core/universal/UScreen.kt

```diff
@@ -127,7 +127,7 @@ abstract class UScreen @JvmOverloads constructor(
     //$$ final override fun tick() = onTick()
     //$$
     //$$ final override fun onClose() {
-    //$$     onScreenClose()
+    //$$     // onScreenClose()
```

Comment:
See notes in pr for reason

Reply:
Is there no other solution? Because `onScreenClose` is the method that the extender would override, and this breaks behavior between versions. Could we instead modify the `onScreenClose` method directly so that it doesn't call to a super method?

Reply:
Yeah, we could remove the ``super.onClose()`` call in ``onScreenClose``, I assume that would fix it, as ``onClose`` is called from that

Reply:
But then again, it works in previous versions so I am unsure as to why it isn't working here

Reply:
I'm coming back to this since I just reviewed the other PR, let's just prevent this cyclical behavior by removing super call in `onScreenClose`, which should keep the desired functionality for all versions.